### DISCUSSION TITLE
1.0 Stable Release is LIVE!

### DISCUSSION THREAD

**Discussion by dayman40** [microsoft/WindowsAppSDK#1806] [Announcements]
## 1.0 stable release of Windows App SDK is now LIVE!  

It's finally here! We've just shipped our latest 1.0 stable release of Windows App SDK! This is the first stable release that fully supports unpackaged apps! In addition, this release includes several new features and lots of bug fixes and stability improvements for WinUI, Windowing, Input, App Lifecycle, Dwrite Core, MRT Core, and deployment.    

We can't wait for you to try all the new features and fixes in the 1.0 stable release.      

## What's new   

The 1.0 stable version of WinUI can be used for shipping production apps. There is also a new high-level windowing API that allows for easy-to-use windowing scenarios that integrates well with the Windows user experience and with other apps. This release of the Windows App SDK is focused on supporting unpackaged apps on x86 and x64. Keep an eye out for ARM64 support, which will be added in the next stable release.   

To see everything that's new, see the [1.0 stable release notes](https://aka.ms/windowsappsdk/1.0-stable/release-notes).   

**To get started:**   

·         [Install developer tools ](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment).    

·         [Create a WinUI 3 app](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app).  

·         [Continue your development journey](https://docs.microsoft.com/en-us/windows/apps/develop).  

## What's next   

To keep up with the Windows App SDK roadmap, vote on features, and see everything we are planning, please see our [productboard portal](https://aka.ms/winappsdkportal), which gets updated regularly.  

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/'>1.0 Stable Release is LIVE! </a>.</em>

**Comment by wjk** [comment]
> We've just shipped our latest 1.0 stable release of Windows App SDK!

Congratulations! Now can you please get to releasing some source code, like you promised a long time ago?

**Comment by castorix** [comment]
**Microsoft.UI.Windowing.AppWindow.GetFromWindowId** now crashes when used with 
**MddBootstrapInitialize**
(tested in C# and VB, it worked with Preview3)

**Comment by DmitryBorodiy** [comment]
What's the point in this if I can't create universal apps with it. WPF is deprecated and I want to use WinUI 3 on all Windows devices including Xbox, Surface Hub and HoloLens. Microsoft, please finally kill WPF! After the release of WinRT and UWP / WinUI no longer need it.

**Comment by DmitryBorodiy** [comment]
UWP is better then classic Desktop apps. When will I be able to see WinUI 3 in UWP? If Microsoft stops supporting UWP, it's probably worth looking for an alternative and leaving the Windows ecosystem.

**Comment by castorix** [comment]
> **Microsoft.UI.Windowing.AppWindow.GetFromWindowId** now crashes when used with **MddBootstrapInitialize** (tested in C# and VB, it worked with Preview3)

I finally fixed it with app.manifest : https://github.com/microsoft/microsoft-ui-xaml/issues/6353

**Comment by DmitryBorodiy** [comment]
I have a bad development experience with WinUI 3.

1. Microsoft needs to revisit WinUI applications.
WinUI 3 apps do not work across the entire Windows ecosystem. Does Panos Panay even know about your WinUI traffic? This something that works on the old WPF platform looks out of date. My favorite Windows and Surface device is the Surface Hub 2S, and you haven't bothered to add support for your ecosystem. Hike Microsoft is not the same company that it used to be. There is no innovation and support for new technologies like UWP. No matter how ridiculous it may sound, you are back to where you started in 2015 and this is a complete failure! You wiped out the Windows mobile marketplace and now take on what's left. 

2. Microsoft should also overhaul Windows 11

Windows 11 is an even bigger problem for Microsoft, it is completely incompatible and has a poor experience with tablets, HoloLens and Surface Hub 2S. At the moment I see no point in number 11 on my devices. 

Microsoft CEO, review your Windows development policy. 

The only thing I liked was the new Microsoft Store, it is really cool compared to the old one, and it is also good that it still remains universal.

**Reply by float34** [reply]
UWP is limited and a bad choice for full-featured desktop apps. So no UWP, please.
